bot/internal/review/review.go

```diff
@@ -197,13 +197,20 @@ func (r *Assignments) getReleaseReviewers() []string {
 	return r.c.ReleaseReviewers
 }
 
-func (r *Assignments) getDocsReviewers(author string) []string {
-	setA, setB := getReviewerSets(author, "Core", r.c.DocsReviewers, r.c.DocsReviewersOmit)
-	reviewers := append(setA, setB...)
+func (r *Assignments) getDocsReviewers(e *env.Environment, files []github.PullRequestFile) []string {
+	// See if any code reviewers are designated preferred reviewers for one of
+	// the changed docs files. If so, add them as docs reviewers.
+	a, b := getReviewerSets(e.Author, "Core", r.c.CodeReviewers, r.c.CodeReviewersOmit)
+	prefCodeReviewers := r.getAllPreferredReviewers(append(a, b...), files)
+
+	// Get the docs reviewer pool, which does not depend on the files
```

Comment:
Can we skip this step if we found enough preferred reviewers in the previous step?

Reply:
That's a good question. For me, it'd be good to keep the reviewer pool wide to increase the likelihood that someone is available to review a docs PR, but that could also mean unnecessary GitHub notification spam. I'm happy to make this change if you think it makes sense. How many reviewers is enough?

Reply:
I'm fine with leaving it as-is.